plugins/simpleadminhacks-paper/src/main/java/com/programmerdan/minecraft/simpleadminhacks/configs/buildlimit/BuildLimitsConfig.java

```diff
@@ -0,0 +1,36 @@
+package com.programmerdan.minecraft.simpleadminhacks.configs.buildlimit;
+
+import com.programmerdan.minecraft.simpleadminhacks.SimpleAdminHacks;
+import com.programmerdan.minecraft.simpleadminhacks.framework.SimpleHackConfig;
+import com.programmerdan.minecraft.simpleadminhacks.framework.utilities.BuildLimit;
+import org.bukkit.configuration.ConfigurationSection;
+
+import java.util.List;
+
+public final class BuildLimitsConfig extends SimpleHackConfig {
+	private BuildLimit[] buildLimits;
+
+	public BuildLimitsConfig(SimpleAdminHacks plugin, ConfigurationSection base) {
+		super(plugin, base);
+	}
+	public BuildLimitsConfig(ConfigurationSection base) {
+		super(SimpleAdminHacks.instance(), base);
+	}
+
+	@Override
+	protected void wireup(ConfigurationSection config) {
+		if (this.shouldEnable()) {
```

Comment:
This shouldEnable() call is still unnecessary and will cause unexpected behaviour if something happens like the hack is disabled in the config, but then is enabled by a command.